elanet/pact/protocol.go

```diff
@@ -13,6 +13,8 @@ const (
 	// DPOSStartVersion is the protocol version which switch to DPOS protocol.
 	DPOSStartVersion uint32 = 20000
 
+	MaxBlockSize = 8000000
```

Comment:
Move this const above to MaxBlocksPerMsg, so they looks like a group.
Add comments to MaxBlockSize.